kahuna/public/js/components/gr-title/gr-title.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+import 'titip';
+
+export const module = angular.module('grTitle', []);
+
+module.directive('grTitle', [function () {
+    return {
+        restrict: 'A',
+        link: function (scope, element, attrs) {
+            const theme = attrs.grTitleTheme || 'default';
+            const position = attrs.grTitlePosition || 'bottom';
```

Comment:
If these aren't used, can we just hardcode the defaults to reduce the API surface?